Code/Framework/AzCore/AzCore/Component/ComponentApplication.cpp

```diff
@@ -1096,7 +1096,7 @@ namespace AZ
     //=========================================================================
     // SetEntityName
     //=========================================================================
-    bool ComponentApplication::SetEntityName(const EntityId& id, const AZStd::string_view name)
+    bool ComponentApplication::SetEntityName(const EntityId& id, AZStd::string_view name)
```

Comment:
suggestion: While I don't disagree with this change we could probably remove it from this PR as it's not very closely related